File: app/src/main/java/com/app/applemvp/mvp/SplashPresenter.java
package com.app.applemvp.mvp;


import com.base.applemvp.common.BasePresenter;



/**
 * applehsp  on 2018/5/17.
 */

public class SplashPresenter extends BasePresenter<SplashView> {


    public SplashPresenter() {
    }




    /**
     * 广告方法
     * @param message
     */
    public void advertList(String message) {
        getView().advertList(message);
    }




}

File: app/src/main/java/com/app/applemvp/mvp/SplashPresenter.kt
package com.app.applemvp.mvp

import com.base.applemvp.common.BasePresenter

/**
 * applehsp  on 2018/5/17.
 */
class SplashPresenter : BasePresenter<SplashView>() {
    /**
     * 广告方法
     * @param message
     */
    fun advertList(message: String) {
        view!!.advertList(message)
    }
}